phpBB/styles/prosilver/theme/common.css

```diff
@@ -1251,6 +1251,15 @@ ul.linklist:after,
 	margin-left: 58px;
 }
 
+.notification-alert,
+.compact .notification-alert {
+	border-radius: 10px;
+	opacity: 0.8;
+	padding: 3px 7px;
+	min-width: 10px;
+	text-align: center;
+}
+
```

Comment:
just use BS code for badge the spacing issues is caused by line-height not being reset. Also to include the pos:rel and top of -3.

```
border-radius: 10px;
	opacity: 0.8;
	display: inline-block;
	padding: 4px 6px;
	font-size: 9px;
	font-weight: 700;
	line-height: 1;
	text-align: center;
	white-space: nowrap;
	vertical-align: baseline;
	margin-left: 4px;
	float: right;
	position: relative;
	top: -3px;
```